### DISCUSSION TITLE
Can't apply edited scheme to profile

### DISCUSSION THREAD

**Discussion by Dan-Essin** [Eugeny/tabby#8193] [Q&A]
Hello,
I prefer a black background so I edited "Material" and changed BG ot #000000. The edited scheme now has a check mark beside it.

When I open a terminal session, the background is still light grey.

The profile's propertis|colors page shows a black background and it has a check mark beside the material profile.

Is this a bug or have I failed to complete some other required step?

Thank you

**Comment by Eugeny** [comment]
If you're not on the latest version, the terminal background might be coming from the window theme - see Settings -> Appearance -> Terminal background
